pegasus/sites.v3/code.org/views/about_headshots.haml

```diff
@@ -5,9 +5,9 @@
       - avatar_url = avatar_image(person[:name_s], 200, true)
       .col-33.avatar_container{:id => person[:id]}
         - unless person[:url_s].nil_or_empty?
-          %a{:href=>person[:url_s], :target=>"_self"}
+          %a{:href=>person[:url_s], :target=>"_blank"}
             - unless avatar_url.nil_or_empty?
-              %img{:src=>avatar_url, :alt=>"", :style=>'max-height: 200px; border-radius: 100px;'}
+              %img{:src=>avatar_url, :alt=>"Open LinkedIn, personal website, or extended bio in a new window to learn more about", :style=>'max-height: 200px; border-radius: 100px;'}
```

Comment:
I assume it's fine to not translate this because employee bios aren't translated?

Reply:
I think that's fine 👍